src/components/AppComponent.js

```diff
@@ -8,7 +8,7 @@ import { handleGestureSegment, handleGestureEnd } from '../shortcuts'
 
 // components
 import { Alert } from './Alert'
-import { Content } from './Content'
+import { ContentContainer } from './Content'
```

Comment:
Let's preserve the name `Content` for the exported component. I would prefer not to change all of the names of the connected components.

Reply:
Ok. Given that we're defining components and containers in the same file, how would you like to differentiate them?
export const **Content** = connect(mapStateToProps, mapDispatchToProps)(**Content**)

Reply:
Good question. What do you think about exporting a `default` instead of a named component?